MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem browseGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[0];
+ImageEditorToolbarItem browseItem = browseGroup.Items.FirstOrDefault();
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+
+{% endhighlight %}
+
+## To get an item from inbuilt bottom toolbar
+
+The `ImageEditorToolbarItem` is placed on the bottom of the image editor, and you can customize the footer toolbar item using the `View` and `Name` properties.
+
+Refer to the following code snippet to customize footer toolbar item.
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem textItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Text");
+
+{% endhighlight %}
+
+## To get an item from sub toolbar
+
+To retrieve an `ImageEditorToolbarItem` from the sub toolbar within the footer toolbar.
+
+Refer to the following code snippet to customize sub items of footer toolbar item.
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem shapeItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Shape");
+ImageEditorToolbar subtoolbar = shapeItem.SubToolbars[0];
+ImageEditorToolbarItem arrowItem = (ImageEditorToolbarItem)subtoolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Arrow");
+
+{% endhighlight %}
+
+N> You can remove the existing toolbar items `Name` from image editor toolbarItems collection based on the index value and change the icon and text values dynamically for any of the already added toolbar item based on the index.
+
+## To change item visibility
+
+You can hide or show the toolbar items by specifying their icon names and setting the boolean values to true or false.
+
+N> You can customize an icon by specifying its `Name`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image.png");
+        imageEditor.SetToolbarItemVisibility("Shape", false);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To change item enable state
+
+You can enable or disable the toolbar items by using `IsEnabled` property in `ImageEditorToolbarItem` class. Specifying their icon names and setting the boolean values to true or false.
```

Comment:
You can enable or disable the toolbar items by using `IsEnabled` property in `ImageEditorToolbarItem` class.

Reply:
Updated